Cargo.toml

```diff
@@ -40,6 +40,7 @@ tempfile = "3.8.1"
 serde_with = "3.4.0"
 once_cell = "1.18.0"
 webbrowser = "0.8.12"
+anyhow = "1.0.75"
```

Comment:
Is this dependency necessary? `eyre` is forked off of anyhow, so you can get a `Result` type from `use color_eyre::eyre::Result`.

Reply:
I didn't know eyre was a fork of anyhow, I'll remove the dependency.